.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: python
 python: "2.7"
+cache:
+  directories:
+    - $HOME/.cache/pip
 install:
-  - pip install ansible ansible-lint
+  - pip install ansible
```

Comment:
Should I rebase on `jessie` branch ? because on `master` ansible version `1.6` is too old ?